packages/block-library/src/verse/index.js

```diff
@@ -2,13 +2,13 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { verse as icon } from '@wordpress/icons';
```

Comment:
I wonder if there's a more generic name for this icon:

![Screenshot 2020-01-23 at 13 39 49](https://user-images.githubusercontent.com/1204802/72985247-db0ba500-3de5-11ea-8dea-5b56a3208642.png)

The general principle is, describe the icon, not the function.

Then there are some gray areas where the icon was created for a particular block, like "page break" and such. 

This one is hard though — perhaps verse is fine, because I can't think of when you'd otherwise use this icon.